docs/preparing-your-app-for-testflight.md

```diff
@@ -0,0 +1,143 @@
+# Preparing your App for TestFlight
+
+Publishing to the App Store requires some preparation. This guide should help you get started with using Transporter and the App Connect site to upload your app to the App Store for internal testing via TestFlight. This guide will use the example of publishing a MacCatalyst app to the Mac App Store, but the process is similar for iOS and tvOS apps.
+
+There is some preparation involved with preparing your Apple Developer account for publishing, such as preparing an App Identifier and a Provisioning Profile. The MAUI documentation has a great guide for that:
+https://aka.ms/maui-publish-app-store
+
+Also, your project will need to have configuration values set in the .csproj file, the Info.plist file, and the Entitlements.plist file.
+
+## Preparing your Project File (.csproj) for App Store submission
+
+Before you can submit your app to the App Store, you need to make sure that it is correctly configured. This includes:
+
+- Setting the build to reference the correct `Entitlements.plist` in your project (For example, one for Debug and one for Release).
+- Setting the correct `ApplicationId` in your project.
+- Setting the correct `ApplicationVersion` in your project.
+- Setting the correct `CodesignKey` in your project.
+- Setting the correct `PackagingSigningKey` in your project.
+- Setting the correct `CodesignEntitlements` in your project.
+- Setting the correct `CodesignProvision` in your project.
+
+### Info.plist
+
+The `Info.plist` file is used by the build system to configure your app and is already generated by your project. It contains info such as the Bundle Display Name and Bundle Identifier.
+### Entitlements.plist
+
+The `Entitlements.plist` file is used by the build system to sign your app.
+
+Some of the values in the `Entitlements.plist` file are set by the build system. These values are:
+
+- com.apple.security.app-sandbox
+- com.apple.security.network.client
+
+## Building your App for Publishing
+
+Once you've set all the correct values, you can build your app for publishing. You can build your app for publishing very easily by using the dotnet build command. Here is an example for building a MacCatalyst app with .NET 7.0:
```

Comment:
"you can build your app for publishing. You can build your app for publishing" - redundant.
Maybe "build your app for publishing. You can do this by using..."